pandas/core/indexes/datetimes.py

```diff
@@ -1937,16 +1937,21 @@ def tz_convert(self, tz):
                      mapping={True: 'infer', False: 'raise'})
     def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         """
-        Localize tz-naive DatetimeIndex to given time zone (using
-        pytz/dateutil), or remove timezone from tz-aware DatetimeIndex
+        Localize tz-naive DatetimeIndex to tz-aware DatetimeIndex.
+
+        This method takes a naive DatetimeIndex object and interprets it as
+        if it is in that timezone. It does not move the time to another
+        timezone.
```

Comment:
I found the interprets word a bit unclear. I think something like "Make a naive DatetimeIndex time zone aware, by attaching a time zone. It does not change the values."

Surely you can write it in a better way, but hopefully you see what I mean.

Reply:
@datapythonista hoes this otherwise I'm going to co[py paste same you suggest above :) 

"This method takes a naive DatetimeIndex object and make this time zone aware. It does not move the time to another timezone.
 Time zone localization helps to switch b/w time zone aware and time zone unaware objects."